### DISCUSSION TITLE
Game minimizes during gameplay

### DISCUSSION THREAD

**Discussion by ThunderVolt-code** [ppy/osu#22921] [Q&A]
### Type

Game behaviour

### Bug description

Game window minimized during gameplay, and could not be reopened until song was failed

### Version

2023.305.0-lazer

### Logs

[input.log](https://github.com/ppy/osu/files/11008994/input.log)
[network.log](https://github.com/ppy/osu/files/11008995/network.log)
[performance.log](https://github.com/ppy/osu/files/11008996/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11008997/runtime.log)
[updater.log](https://github.com/ppy/osu/files/11008998/updater.log)
[database.log](https://github.com/ppy/osu/files/11008999/database.log)

**Comment by bdach** [comment]
Please upload a video of this.
